src/ray/gcs/redis_context.cc

```diff
@@ -135,7 +135,30 @@ RedisContext::~RedisContext() {
   }
 }
 
-Status RedisContext::Connect(const std::string &address, int port, bool sharding) {
+Status authenticate_redis(redisContext *context, const std::string &password) {
```

Comment:
These functions don't follow the naming convention I think. @pcmoritz can comment, but I think it should be `AuthenticateRedis`.